tensorflow.sh

```diff
@@ -1,14 +1,22 @@
-driver_version=367.48
-cuda="cuda-linux64-rel-8.0.44-21122537.run"
-cudnn="cudnn-8.0-linux-x64-v5.1.tar"
+driver_version=367.44
```

Comment:
I believe the source of #3 is that we need to be using the exact same driver version as on sherlock, 367.48 .  So I actually would keep this line and the next 2 the same.